src/RedisIPCMessenger.ts

```diff
@@ -110,6 +110,10 @@ export default class RedisIPCMessenger implements IPCMessenger {
         throw new Error(`Failed to map ${key} to a callback`);
       }
 
+      if (instance === this.instance) {
```

Comment:
Also, aren't we comparing a `string` to an `Instance` here?

Reply:
Correct, fixed